src/Language/PureScript/Interactive/Types.hs

```diff
@@ -25,10 +30,15 @@ data PSCiState = PSCiState
   { psciImportedModules     :: [ImportedModule]
   , psciLetBindings         :: [P.Declaration]
   , psciLoadedExterns       :: [(P.Module, P.ExternsFile)]
+
+  -- These are derived from updateImportExports each time a module is imported,
+  -- a let binding is added, or the session is cleared or reloaded
+  , psciImports             :: P.Imports
+  , psciExports             :: P.Exports
   } deriving Show
 
 initialPSCiState :: PSCiState
-initialPSCiState = PSCiState [] [] []
+initialPSCiState = updateImportExports (PSCiState [] [] [] nullImports nullExports)
```

Comment:
Will `updateImportExports` do anything here?

Reply:
Yes, perhaps surprisingly. It adds the types from `Prim`, which is implicitly imported in the module created in that function.